wmn-data.json

```diff
@@ -6560,5 +6560,15 @@
         "known" : ["carlito", "taylor"],
         "cat" : "social"
       }
+      {
+        "name" : "Letterboxd",
+        "uri_check" : "https://letterboxd.com/{account}",
+        "e_code" : 200,
+        "e_string" : "RECENT ACTIVITY",

```

Comment:
I tried via web browser and curl and cannot see the `RECENT ACTIVITY` `e_string` in any response. I get a good response but don't see that string. I'd suggest using the string `’s profile on Letterboxd` which appears in the title tag.